delegation/src/Pos/Delegation/Helpers.hs

```diff
@@ -38,6 +39,6 @@ isRevokePsk = isSelfSignedPsk
 dlgMemPoolApplyBlock :: MainBlock -> DlgMemPool -> DlgMemPool
 dlgMemPoolApplyBlock block m = flip execState m $ do
     let (toDelete,toReplace) =
-            partition isRevokePsk (getDlgPayload $ block ^. mainBlockDlgPayload)
+            partition isRevokePsk (toList $ getDlgPayload $ block ^. mainBlockDlgPayload)
```

Comment:
It would probably make sense to define `ToList` (or `Container` in old `universum`) for `DlgPayload`.